lib/delivery_service.rb

```diff
@@ -1,36 +1,69 @@
+# frozen_string_literal: true
+
 require_relative 'car'
 require_relative 'bike'
 
+# class DeliveryService
 class DeliveryService
-  attr_reader :cars, :bikes
+  attr_reader :cars, :bikes, :autopark
 
   def initialize
     @cars  = Array.new(Constants::CARS[:quantity]) { Car.new }
     @bikes = Array.new(Constants::BIKES[:quantity]) { Bike.new }
+    @autopark = @cars + @bikes
```

Comment:
As I can see you use @cars and @bikes instance variables only for @AutoPark. I don`t like this idea
You can do something like
@AutoPark = []
Constants::BIKES[:quantity].times { autopark << Bike.new }
And remove then attr_reader :cars, :bikes you need only :autopark

Reply:
done